locales/fi.schema.json

```diff
@@ -610,7 +610,7 @@
             "label": "Muotokuva"
           },
           "options__3": {
-            "label": "Neliö"
+            "label": "Square"
```

Comment:
It looks like we are still getting this issue 🤔 @ludoboludo I see your comment above. Do you have context on next steps?

Reply:
I had shared the issue with the localization team and they were looking into it. I followed up with them today to mention that it's still an issue 👍